python/pyspark/sql/tests.py

```diff
@@ -3920,6 +3920,14 @@ def test_vectorized_udf_null_string(self):
         res = df.select(str_f(col('str')))
         self.assertEquals(df.collect(), res.collect())
 
+    def test_vectorized_udf_string_in_udf(self):
+        from pyspark.sql.functions import pandas_udf, col
+        import pandas as pd
+        df = self.spark.range(10)
+        str_f = pandas_udf(lambda x: pd.Series(["%s" % i for i in x]), StringType())
```

Comment:
Not a big deal. How about `pd.Series(map(str, x))`?

Reply:
Sounds good. I'll take it.